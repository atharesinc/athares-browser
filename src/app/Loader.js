import React from "react";

const Loader = props => {
	return (
		<div className="loader-wrapper">
			<img
				className="loader-img"
				src="https://s3.us-east-2.amazonaws.com/athares-images/Athares-logo-small-white-no-circle.png"
				alt="Loading"
			/>
			<div className="loader" />
			<div className="loader delay" />
		</div>
	);
};
export default Loader;
