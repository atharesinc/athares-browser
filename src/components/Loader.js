import React from "react";

const Loader = props => {
	return (
		<div className="loader-wrapper">
			<img
				className="loader-img"
				src="/img/Athares-logo-small-white-no-circle.png"
				alt="Loading"
			/>
			<div className="loader" />
			<div className="loader delay" />
		</div>
	);
};
export default Loader;
