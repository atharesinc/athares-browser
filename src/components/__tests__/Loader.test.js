import { create } from "react-test-renderer";
import Loader from "../Loader.js";
import React from "react";

describe("Loader component", () => {
  it("should render loader component", () => {
    const tree = create(<Loader />);

    expect(tree).toMatchSnapshot();
  });
});
