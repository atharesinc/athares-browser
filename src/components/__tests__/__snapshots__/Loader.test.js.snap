// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render loader component 1`] = `
<div
  className="loader-wrapper"
>
  <img
    alt="Loading"
    className="loader-img"
    src="/img/Athares-logo-small-white-no-circle.png"
  />
  <div
    className="loader"
  />
  <div
    className="loader delay"
  />
</div>
`;
